docassemble/HousingCodeChecklist/data/questions/customized_screens.yml

```
@@ -266,6 +266,34 @@ under: |
   ${x}
 progress: 99
 ---
+id: signature choice
+decoration: file-signature
+question: |
+  Your documents are almost ready
+subquestion: |
+  How do you want to sign them?
+fields: 
+  - I will sign: signature_choice
+    input type: radio
+    choices:
+      - On a phone with my finger: phone
+        help: |
+          We can text or share a link with your phone on the next screen
+      - On a computer with my mouse: this_device
+      - On the paper with a pen after I print the documents: sign_after_printing
+  - I am signing under the pains and penalties of perjury: acknowledgement
+    datatype: checkboxes
+    none of the above: False    
+    minlength: 1
+    choices:
+      - Acknowledge.
+    validation messages:
+      minlength: |
+        You cannot continue unless you acknowledge.
```

Let's use the language from AssemblyLine (https://github.com/SuffolkLITLab/docassemble-AssemblyLine/blob/1565ac4c57c2b88380ef25911587f78ad5018d17/docassemble/AssemblyLine/data/questions/ql_baseline.yml#L1196)

```suggestion  
  - no label: users[0].states_above_true
    datatype: checkboxes
    minlength: 1
    choices:
      - I affirm that the answers I have given are true, under the pains and penalties of perjury: states_true
    validation messages:
      minlength: |
        You cannot keep going unless you did your best to tell the truth.
```